### PR TITLE
fwconsole: add page

### PR DESCRIPTION
**Pull request by aqeelabpro**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 16.0.40.11

### CODE REVIEW

File: pages/linux/fwconsole.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`fwconsole start`
+
+- Stop Asterisk and run other needed FreePBX commands:
+
+`fwconsole stop`
+
+- View and update settings:
+
+`fwconsole setting {{keyword}} {{newvalue}}`
```
Comment (spageektti) [inline pages/linux/fwconsole.md:20]:
```suggestion
`fwconsole setting {{keyword}} {{new_value}}`
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution. I wish a wonderful christmas time!

File: pages/linux/fwconsole.md (hunk 1 above)
Comment (aqeelabpro) [reply pages/linux/fwconsole.md]: Hi, I have committed to the changes you suggested. Please let me know if anything else needs to be updated. Thanks for the feedback!

File: pages/linux/fwconsole.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# fwconsole
+
+> Manage and configure your FreePBX system (PBX server).
+> More information: <https://wiki.freepbx.org>.
```
Comment (spageektti) [inline pages/linux/fwconsole.md:4]:
```suggestion
> More information: <https://sangomakb.atlassian.net/wiki/spaces/FP/overview>.
```
the link you provided redirects to this site

Comment (spageektti) [reply pages/linux/fwconsole.md:4]: Or maybe use this one?
https://sangomakb.atlassian.net/wiki/spaces/PG/pages/41746505/fwconsole+commands+15

File: pages/linux/fwconsole.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- List available backups:
+
+`fwconsole backup --list`
+
+- List available FreePBX commands:
+
+`fwconsole list`
+
+- Change ownership of files:
```
Comment (spageektti) [inline pages/linux/fwconsole.md:30]:
```suggestion
- Change ownership of all files and directories that FreePBX needs to be owned by the apache user:
```

File: pages/linux/fwconsole.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# fwconsole
+
+> Manage and configure your FreePBX system (PBX server).
+> More information: <https://wiki.freepbx.org>.
+
+- Reload FreePBX configurations:
+
+`fwconsole reload`
+
+- Start Asterisk and run other needed FreePBX commands:
```
Comment (spageektti) [inline pages/linux/fwconsole.md:10]:
```suggestion
- Start Asterisk and other commands needed by FreePBX:
```

File: pages/linux/fwconsole.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`fwconsole start`
+
+- Stop Asterisk and run other needed FreePBX commands:
```
Comment (spageektti) [inline pages/linux/fwconsole.md:14]:
```suggestion
- Stop Asterisk and other commands needed by FreePBX:
```

File: pages/linux/fwconsole.md (hunk 2 above)
Comment (aqeelabpro) [reply pages/linux/fwconsole.md:4]: When I was testing locally, I added this https://sangomakb.atlassian.net/wiki/spaces/PG/pages/41779247/fwconsole+commands+13
but I thought it was a long URL, so I just added the <https://wiki.freepbx.org>, which directs to the same website, ok, I will change it to https://sangomakb.atlassian.net/wiki/spaces/PG/pages/41779247/fwconsole+commands+13

**Review by aqeelabpro — COMMENTED** [review]
Thank you for your suggestion! I've made the necessary changes to the link based on your feedback.
Apologies for any confusion earlier—this is my first contribution, and I’m learning as I go.
Please let me know if there’s anything else I can improve.

**Review by aqeelabpro — COMMENTED** [review]
Resolved the suggested changes

**Review by spageektti — APPROVED** [review]

**Pull request merged**